lib/snmp/src/agent/snmpa.erl

```diff
@@ -265,34 +265,26 @@ symbolic_store, local_db and master_agent (and sub-agents).
   Sets verbosity for all sub-agent(s) controlled by this (master) agent.
 
 - **`master_agent | pid()`** - Sets verbosity for the agent process.
+
+The following text documents expected input-output relations
+
+- If `Target :: all | net_if | note_store | mib_server | symbolic_store | local_db`,
+  then `Verbosity :: snmp:verbosity()`.
+
+- If `Target :: master_agent`,
+  then `Verbosity :: {subagents, snmp:verbosity()}`
+
+- If `Target :: pid() | atom()`,
+  then `Verbosity :: snmp:verbosity() | {subagents, snmp:verbosity()}`.
 """.
 -spec verbosity(Target, Verbosity) -> snmp:void() when
-      Target    :: all,
-      Verbosity :: snmp:verbosity();
-               (Target, Verbosity) -> snmp:void() when
-      Target    :: net_if,
-      Verbosity :: snmp:verbosity();
-               (Target, Verbosity) -> snmp:void() when
-      Target    :: note_store,
-      Verbosity :: snmp:verbosity();
-               (Target, Verbosity) -> snmp:void() when
-      Target    :: mib_server,
-      Verbosity :: snmp:verbosity();
-               (Target, Verbosity) -> snmp:void() when
-      Target    :: symbolic_store,
-      Verbosity :: snmp:verbosity();
-               (Target, Verbosity) -> snmp:void() when
-      Target    :: local_db,
-      Verbosity :: snmp:verbosity();
-               (Target, Verbosity) -> snmp:void() when
-      Target    :: master_agent | Agent,
-      Agent     :: pid() | AgentName,
-      AgentName :: atom(),
-      Verbosity :: {subagents, snmp:verbosity()};
-               (Agent, Verbosity) -> snmp:void() when
-      Agent     :: pid() | AgentName,
-      AgentName :: atom(),
-      Verbosity :: snmp:verbosity().
+      Target :: Actions | Agent | Pid,
+      Actions :: all | net_if | note_store | mib_server | symbolic_store | local_db,
```

Comment:
All these ('Actions') are "logical names" for different processes in the agent.
'net_if' is the network interface process, for instance.
I would prefer 'LogicalName' for these.
So, 'Target' would something like: 'all' | LogicalName | Agent | pid()
And LogicalName = net_if | note_store ...

Reply:
Fixed in the latest commit.
There is one thing that is not clear to me. If I pass as input:

```
verbosity( PidOrAgentName :: pid() | atom(),
         Verbosity :: snmp:verbosity() ) -> ????
```

what do I expect from the original types (before this change)?
From reading the docs before the change, I would expect `snmp:verbosity() | {subagents, snmp:verbosity()}`.
Is this correct? Just checking that this is expected before trying to correct the types

Reply:
PidOrAgentName is pid() or 'master_agent'
'atom()' is a registerd name (such as snmp_master_agent)
The return value should be typed as 'snmp:void()'. If the call targets a *single entity*
(such as net_if), it will in practice be the old verbosity.
'{subagents, ...}' will only work if the target is the "master agent" (regardless how
it was addressed; 'master_agent' | pid() | "registered name").
It is of course possible to target a subagent directly if you know it's pid.